clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/provider/agent/DcosServerGroupCachingAgent.groovy

```diff
@@ -374,14 +388,14 @@ class DcosServerGroupCachingAgent implements CachingAgent, AccountAware, OnDeman
     log.info("Caching ${cachedInstances.size()} instances in ${agentType}")
 
     new DefaultCacheResult([
-                             (Keys.Namespace.SERVER_GROUPS.ns) : cachedServerGroups.values(),
```

Comment:
In general, to keep these more reviewable please separate formatting & functionality changes.